x/ibc-transfer/keeper/relay.go

```diff
@@ -12,19 +12,20 @@ import (
 
 // SendTransfer handles transfer sending logic. There are 2 possible cases:
 //
-// 1. Sender chain is the source chain of the coins (i.e where they were minted): the coins
-// are transferred to an escrow address (i.e locked) on the sender chain and then
-// transferred to the destination chain (i.e not the source chain) via a packet
-// with the corresponding fungible token data.
+// 1. Sender chain is acting as the source zone. The coins are transferred to an
+// escrow address (i.e locked) on the sender chain and then transferred to the
+// receiving chain through IBC TAO logic. It is expected that the receiving
+// chain will mint vouchers to the receiving address.
 //
-// 2. Coins are not native from the sender chain (i.e tokens sent where transferred over
-// through IBC already): the coins are burned and then a packet is sent to the
-// source chain of the tokens.
+// 2. Sender chain is acting as the sink zone. The coins (vouchers) are burned
+// on the sender chain and then transferred to the receiving chain though IBC
+// TAO logic. It is expected that the receiving chain will unescrow the fungible
+// token and send it to the receiving address.
```

Comment:
nit, I'd use your forward/backward in the timeline here to briefly explain what source and sink zones are.